frontend/src/pages/Overview/ProjectDialogContent.js

```diff
@@ -29,18 +28,21 @@ const ProjectDialogContent = props => {
       <Divider />
       <div>
         <Budget
-          currencyTitle={strings.project.project_currency}
-          currency={props.projectToAdd.currency}
-          budgetLabel={strings.project.project_budget_amount}
-          budgetHintText={strings.project.project_budget_amount_description + " " + toAmountString(99999.99)}
-          budget={props.projectToAdd.amount}
-          storeProjectedBudget={props.storeProjectedBudget}
-          organization={props.projectToAdd.organization}
+          isProjectBudget={true}
```

Comment:
Without `isProjectBudget `and `isSubprojectBudget`, the code is less readable and useable, because:

- if on project-level: The user should always be able to type an organization in a Textfield
- if on subproject-level: The user should be able to select an organization in a Dropdown



The old solution was to check if projectProjectedBudgets is undefined (then it is project-level => Textfield for organization) or an array (then it is subproject-level => Dropdown to select organization.


The new solution:
  // If no project budget has been set, no subproject budget can be set
  // Project budgets can always be set
  const canSetNewBudget = (isSubprojectBudget && !_isEmpty(projectProjectedBudgets)) || isProjectBudget;

Reply:
done